### DISCUSSION TITLE
Feature Request: Rolling Deletes

### DISCUSSION THREAD

**Discussion by mcnuttandrew** [umami-software/umami#644] [Ideas]
Is it possible to configure the app to delete events that occurred before a certain period of time? For instance, say I only care about the last 40 days of data, would it be possible to delete all data that occurred before that? I know this would be possible by just writing sql, but I think it'd be a useful feature to have built directly into the application.

**Comment by mikecao** [comment]
Well it can't do it automatically. You would have to enter the app and click something manually. Although, I'm not really sure if it's a feature worth having since it can be scripted elsewhere and can be run automatically, like a cron script.
